Disable broken test_cuda_kernel_loop_overflow_large test

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29904 Disable broken test_cuda_kernel_loop_overflow_large test**

See https://github.com/pytorch/pytorch/issues/26838

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D18539740](https://our.internmc.facebook.com/intern/diff/D18539740)

ezyang committed:
Disable broken test_cuda_kernel_loop_overflow_large test

See https://github.com/pytorch/pytorch/issues/26838

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

kostmo commented:
## CircleCI build failures summary
As of commit 7234f63:
* **1/1** failures introduced in this PR
* **0/1** recognized as flaky



Here are the [reasons each build failed](https://dr.pytorch.org/commit-details.html?sha1=7234f631a5c97289cdbb2ce8616338e77d39f744):

| Job | Step | Log excerpt |
| --- | --- | --- |
| pytorch_linux_xenial_py3_6_gcc5_4_test | Test | `main()\':\n/tmp/torch_extensions/test_compilation_error_formatting/main.cpp:2:23: error:` |


---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

ezyang committed:
Update on "Disable broken test_cuda_kernel_loop_overflow_large test"


See https://github.com/pytorch/pytorch/issues/26838

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

Differential Revision: [D18539740](https://our.internmc.facebook.com/intern/diff/D18539740)

[ghstack-poisoned]
